targets/targets.json

```diff
@@ -8011,7 +8012,7 @@
         "extra_labels_remove": ["CORDIO"],
         "components_add": ["SPM_MAILBOX"],
         "macros_add": ["PSOC6_DYNSRM_DISABLE=1", "MBEDTLS_PSA_CRYPTO_C"],
-        "hex_filename": "psa_release_1.0.hex",
+        "hex_filename": "mbed_os.hex",
```

Comment:
I don't mean to bikeshed this name too much, but is there a more descriptive name than `mbed_os.hex` that can be used?

Reply:
Fixed